src/components/card/Card.js

```diff
@@ -1,20 +1,31 @@
 import React from 'react';
 import { Heading, Box, Text } from '@chakra-ui/react';
-const Card = ({ book }) => {
+import { Skeleton } from '@chakra-ui/react';
+const Card = ({ book, books }) => {
   const { title, authors, line } = book;
+
   return (
-    <Box boxShadow="base" p={5} borderRadius="lg" _hover={{boxShadow: "md"}}>
-      <Heading as="h4" size="sm">{title}</Heading>
-      <Text mt={3} mb={3}>
-        <Text as="strong">Auteurs: </Text> 
-        {authors}
-      </Text>
-      
-      <Text>
-        <Text as="strong">Note: </Text> 
+    <Skeleton isLoaded>
```

Comment:
Thanks @opethrocks for your contribution.

I'm not sure I see the changes in UI brought by this modification. 

Can you please make us a screenshot/small gif video/animation so that I can visualize it ? Thanks

Reply:
https://gfycat.com/readyslimylacewing
https://gfycat.com/yawningfabulousaustralianfurseal

As you can see from the first recording, the skeleton adds a fade in animation to the card component. I would like to actually see the skeleton animation for second, but I can't figure out how. The isLoaded prop set to false will show the skeleton animation indefinitely. I have set it to the prop to book as I thought it would be null until it receives a response from the api.